docs/storage/rs3/setup.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: RS3 Setup
+slug: /storage/rs3/setup
+sidebar_position: 0
+description: Instructions for getting started using RS3 Storage
+---
+
+:::note Overview
+
+These instructions take you through the steps required to configure your application to use RS3 storage.
+
+__Note__: You will need to setup your application to use Responsive, and configure it with an SDK license. Instructions on how to do this can be found in the [migration guide](/sdk/migrate).
+
+__Note__: You will need to have an RS3 cluster deployed and determine your RS3 endpoint. RS3 is currently in private beta. If you would like to join, apply [here](https://www.responsive.dev/rs3-storage).
```

Comment:
nit: let's just point people to sign up to our cloud, it's not private beta it's just manual provisioning (which is mentioned in product)